ONSdigital/eq-survey-runner
# Renaming model to schema

### What is the context of this PR?

Renaming model to schema and adding the new questionnaire state ready for the refactoring work
### How to review
1. Check out this branch
2. Check all tests run
3. Complete a survey and check nothing is broken
### Who worked on the PR

@warrenbailey
